azure-pipelines.yml

```diff
@@ -534,9 +534,34 @@ jobs:
   - bash: |
       set -e
       . util/build_consts.sh
-      ci/scripts/run-fpga-cw310-tests.sh cw310_test_rom || { res=$?; echo "To reproduce failures locally, follow the instructions at https://docs.opentitan.org/doc/getting_started/setup_fpga/#reproducing-fpga-ci-failures-locally"; exit "${res}"; }
+      ci/scripts/run-fpga-cw310-tests.sh cw310_test_rom,-jtag || { res=$?; echo "To reproduce failures locally, follow the instructions at https://docs.opentitan.org/doc/getting_started/setup_fpga/#reproducing-fpga-ci-failures-locally"; exit "${res}"; }
     displayName: Execute tests
 
+- job: execute_rom_fpga_jtag_tests_cw310
+  displayName: Experimental CW310 ROM JTAG Tests
+  pool: FPGA
+  timeoutInMinutes: 5
```

Comment:
Five minutes may have been overly optimistic.

Reply:
Indeed! Just bumped the timeout up to 45 minutes to match the sibling jobs. I'll use the actual runtime to inform the limit this time instead of guessing :)